K8s-deployment/K8s-cluster/addons/mon-stack/prometheus/prometheus-values.yaml

```diff
@@ -1564,21 +1746,23 @@ serverFiles:
           - action: labelmap
             regex: __meta_kubernetes_service_label_(.+)
           - source_labels: [__meta_kubernetes_namespace]
-            target_label: kubernetes_namespace
```

Comment:
keep ``target_label`` as before for 
- kubernetes_namespace
- kubernetes_name
- kubernetes_node